naloge/maphija.tex

```diff
@@ -25,5 +25,59 @@
 Zakaj?
 \end{vprasanje}
 \begin{odgovor}
+
+Definiramo si spremenljivke, ki jih bomo potrebovali in izpišemo znane podatke.
+\begin{itemize}
+\item[] $X$ \dots zaslužek
+\item[] $A$ \dots recept uspe
+\item[] $B$ \dots prosijo za nasvet
+\item[] $C$ \dots svetujejo ugodno
+\item[] $D$ \dots sami proizvajajo
+\end{itemize}
+\begin{align*}
+&P(A) = \frac{2}{5} && P(\neg A) = \frac{3}{5}\\
+&P(C|A) = \frac{3}{4} & &P(C|\neg A) = \frac{1}{2}\\
+&P(\neg C|A) = \frac{1}{4} & &P(\neg C| \neg A) = \frac{1}{2}
+\end{align*}
+Izračunajmo verjetnosti, ki jih bomo v nalogi potrebovali.
+\begin{align*}
+&P(C) = P(C|A) P(A) + P(C|\neg A) P(\neg A) =\frac{3}{4}\cdot \frac{2}{5} + \frac{1}{2}\cdot \frac{3}{5} = \frac{3}{5}\\
+&P(A|C) = \frac{P(C|A)P(A)}{P(C)} = \frac{3}{4}\cdot \frac{2}{5}\cdot \frac{5}{3}=\frac{1}{2}\\
+&P(\neg A|C) = \frac{P(C|\neg A)P(\neg A)}{P(C)} = \frac{1}{2}\cdot \frac{3}{5}\cdot \frac{5}{3} = \frac{1}{2}
+\end{align*}
+
+\begin{itemize}
+\item[kavarna ne prosi za svetovanje]
```

Comment:
Take oznake ne izgledajo najlepše, saj segajo preko levega roba besedila skoraj do levega roba strani. Namesto opisne rešitve bi svetoval, da naredita odločitveno drevo - v besedilu torej poračunata le verjetnosti in zaslužke v listih, vsi ostali izračuni in odločitve pa so implicitni na drevesu. Poglejta si, kako je to narejeno pri drugih rešitvah (tudi obliko drevesa si lahko izposodita od katere druge naloge).